app/assets/javascripts/short-form/templates/e6a-right-to-return-preference.html.slim

```diff
@@ -0,0 +1,39 @@
+.app-card.form-card
```

Comment:
This was app/assets/javascripts/short-form/templates/e6a-alice-griffith-preference.html.slim before but I renamed it and the diff was big enough that github didn't pick it up as a rename